Exiled.API/Features/Plugin.cs

```diff
@@ -167,6 +167,9 @@ public Plugin()
             InternalTranslation = new TTranslation();
         }
 
+        /// <inheritdoc/>
+        public override Assembly Assembly { get; } = Assembly.GetCallingAssembly();
```

Comment:
Can't the assembly be set in the constructor?

Reply:
Maybe. I can try doing it. Rider was yelling at me before though.

Reply:
Just tested, it's working fine in the constructor.